packages/docs/components/SideNav/styled.tsx

```diff
@@ -2,7 +2,7 @@ import { Flex } from '@bigcommerce/big-design';
 import styled from 'styled-components';
 
 export const StyledFlex = styled(Flex)`
-  ${({ theme }) => theme.shadow.raised}
+  ${({ theme }) => theme.shadow.raised};
```

Comment:
🍹 You shouldn't need to add a semi-colon here. `theme.shadow.raised` already outputs semi's on each CSS declaration already.
```suggestion
  ${({ theme }) => theme.shadow.raised}
```